AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.ts

```diff
@@ -25,6 +31,8 @@ export class AllocateHearingsComponent implements OnInit {
             caseNumber: [''],
             isUnallocated: [false]
         });
+        const today = new Date();
+        this.todayDate = this.datePipe.transform(today, 'yyyy-MM-dd');
```

Comment:
See other comment re: date pipe in template

Reply:
see reply above

Reply:
I have now refactored the code using date pipe in the html